docs/sources/whatsnew/whats-new-next/index.md

```diff
@@ -18,6 +18,56 @@ weight: -37
 
 Welcome to Grafana Cloud! Read on to learn about the newest changes to Grafana Cloud.
 
+## Quicker path to troubleshooting in Kubernetes Monitoring
+
+<!-- Beverly Buchanan --> 
+
+_Generally available in Grafana Cloud_
+
+You can use Kubernetes Monitoring to more quickly identify and troubleshoot issues with these views:
+
+- The home page shows a [snapshot of issues](/docs/grafana-cloud/monitor-infrastructure/kubernetes-monitoring/navigate-k8s-monitoring/#see-the-issues-at-a-glance) that reach a specific threshold. 
+  
+  (ADD SCREENSHOT AFTER CROW's NEST LANGUAGE REMOVED)
+
+  This view shows:
+
+  - Pods that have been in a non-running state for more than 15 minutes
+  - Nodes with CPU usage above 90% for more than five minutes
+  - Nodes using more than 90% of memory for more than five minutes
+  - Persistent Volumes with capacity above 90%
```

Comment:
I assume "Volumes" is intentionally capitalized here?

Reply:
Yes